registration/models.py

```diff
@@ -422,7 +422,7 @@ def _activate(self, profile, get_profile):
         """
 
         if not profile.user.is_active and not profile.activated:
-            site = Site.objects.get_current()
+            site = get_current_site()
```

Comment:
If `SupervisedRegistrationManager._activate` interface is changed (like `_activate(self, site, profile, get_profile)`), have to change `RegistrationManager._activate` and `RegistrationManager.activate_user` as same as `SupervisedRegistrationManager`.

Its not have `requests` or `site`.